modules/core/arrow-data/src/test/kotlin/arrow/data/ValidatedTest.kt

```diff
@@ -1,41 +1,35 @@
 package arrow.data
 
 import arrow.core.*
+import arrow.instances.*
+import arrow.test.UnitSpec
+import arrow.test.laws.*
+import arrow.typeclasses.Eq
+import arrow.typeclasses.Semigroup
 import io.kotlintest.KTestJUnitRunner
 import io.kotlintest.matchers.fail
 import io.kotlintest.matchers.shouldBe
-import io.kotlintest.matchers.shouldNotBe
-import arrow.instances.*
-import arrow.syntax.applicative.map
-import arrow.syntax.validated.valid
 import org.junit.runner.RunWith
-import arrow.test.UnitSpec
-import arrow.test.laws.*
-import arrow.typeclasses.*
 
 @RunWith(KTestJUnitRunner::class)
 class ValidatedTest : UnitSpec() {
 
     init {
 
-        "instances can be resolved implicitly" {
-            functor<ValidatedPartialOf<String>>() shouldNotBe null
-            applicative<ValidatedPartialOf<String>>() shouldNotBe null
-            foldable<ValidatedPartialOf<String>>() shouldNotBe null
-            traverse<ValidatedPartialOf<String>>() shouldNotBe null
-            applicativeError<ValidatedPartialOf<String>, String>() shouldNotBe null
-            eq<Validated<String, Int>>() shouldNotBe null
-            show<Validated<String, Int>>() shouldNotBe null
-        }
+        val EQ = Validated.eq(StringEqInstance, IntEqInstance)
+
+        val VAL_AP = Validated.applicative<String>(StringMonoidInstance)
```

Comment:
`Val_AP` is not a good naming convention to use in FP or Kotlin, `ValidatedAp` or `validatedApplicative` are probably better names for this

Reply:
We've been using cryptic names on tests so far, it's 'aight